contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,141 @@
+
+1. Increase the number of high-quality project ideas.
+2. Make your engineers own their experiments.
+
+### Increase the number of high-quality project ideas
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
+
+### Make your engineers own their experiments
+
+Every engineer must be responsible for making their experiments as impactful as possible – even if it wasn't originally their idea. 
+
+Empowerment is the key here. When engineers take charge of their experiments, they become more invested in the outcome, leading to better results.
+
+In practice, this means:
+
+- Engineers work to improve the initial hypothesis.
+- Engineers develop further iterations to test. 
+- Removing processes that prevent engineers from making quick decisions on their own.
+- Allowing engineers space to fail without being harsh on them.
+
+## Problem 4: Your team is afraid of making mistakes
+
+If your team is not making mistakes, you're not moving fast enough. 
+
+In fact, it's not uncommon for growth teams to cause a few incidents! And their goal must not be to avoid them.
+
+"When airbags were [introduced](https://www.purdue.edu/uns/html4ever/2006/060927ManneringOffset.html) in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they resolve them quickly."
+
+### How to fix it
+
+Your aim should be to build a resilient system that can rapidly recover from mistakes or incidents.
+
+In the context of growth teams, there are a few ways to do this:
+
+- Monitor critical metrics and set up alerts. Ensure you can quickly roll back changes using [configurators or feature flags](https://posthog.com/blog/feature-flags-vs-configuration).
+- Set up [review apps](https://seanconnolly.dev/review-applications) so your team can easily test and dogfood your experiments.
+- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.
+- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
```

Comment:
```suggestion
- Monitor critical metrics and set up alerts. Ensure you can quickly roll back changes using [feature flags](/blog/feature-flags-vs-configuration).

- Set up [review apps](https://seanconnolly.dev/review-applications) so your team can easily test and dogfood your experiments.

- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.

- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
```